vassal-app/src/main/java/VASSAL/counters/GlobalCommandTarget.java

```diff
@@ -67,6 +68,34 @@ public Configurer getConfigurer(AutoConfigurable c, String key, String name) {
     DECK     // {@link VASSAL.build.module.map.DeckGlobalKeyCommand}
   }
 
+  /**
+   * Comparison Modes for property match
+   */
+  public enum CompareMode {
+    EQUALS("=="),
+    NOT_EQUALS("!="),
+    GREATER(">"),
+    GREATER_EQUALS(">="),
+    LESS("<"),
+    LESS_EQUALS("<=");
```

Comment:
What about =~ and !~ ?

Reply:
Oh right - I had stared at PropertiesPieceFilter for a while and couldn't decipher the regex mechanism. So I have resorted to "attempting to learn it from google", with results now pushed. See what you think & give it a test -- I never end up using this in my modules because it always takes me too many tries to get the syntax right (among other things my brain always thinks it will be ~= which I literally had to look up *three times* while coding this I guess I could have snuck in a ~= synonym and forgotten about it in peace, haha), but anyway I don't have any convenient test code for it.